### PR TITLE
refactor: replace bundler from react-script to Vite

### PR DESCRIPTION
**Pull request by therealemjy**
- replace bundler, from `react-scripts` to Vite
- update `react` to v18
- use local version of `eslint` to prevent issues with using global version installed on client
- upgrade `react-query` to v5

### CODE REVIEW

**Review by gleiser-oliveira — COMMENTED** [review]
I've noticed the token logos are missing, with the src tag looking like this: https://landing-git-refactor-bundler-venus-protocol.vercel.app/assets/undefined

Screen Shot 2024-06-21 at 9 35 17

**Comment by therealemjy** [comment]
Good catch, I completely missed that. I think I'm incorrectly mapping to the public folder, I'll get this fixed. Thanks!

**Comment by therealemjy** [comment]
It's fixed.

File: src/api/constants.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 export const getTokenPublicUrl = (filePath: string) =>
-  `${process.env.PUBLIC_URL}/coins/${filePath}`;
+  `/coins/${filePath}`;
+
+
+console.log(import.meta.url);
```
Comment (gleiser-oliveira) [inline src/api/constants.ts:5]: Log here

**Review by gleiser-oliveira — APPROVED** [review]

**Pull request merged**